test/setup/iam.tf

```
@@ -79,6 +79,7 @@ resource "google_billing_account_iam_member" "int_test" {
 }
 
 resource "google_service_account_iam_member" "cloud_build_impersonation" {
+  count              = var.build_project_number == null ? 0 : 1
   service_account_id = google_service_account.int_test.name
```

```suggestion
resource "google_service_account_iam_member" "cloud_build_impersonation" {
  count = var.build_project_number == null ? 0 : 1
  
  service_account_id = google_service_account.int_test.name
```

add an empty line between the meta-arguments and the arguments
https://developer.hashicorp.com/terraform/language/style#code-formatting